Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/docrepository/adapter/AdapterComponentDocRepositoryHelper.java

```diff
@@ -248,23 +237,22 @@ String extractMetadataFromSlots(List<oasis.names.tc.ebxml_regrep.xsd.rim._3.Slot
      * @return Returns the value of the first metadata value with the given metadata name. Null if not present.
      */
     String extractPatientInfo(List<oasis.names.tc.ebxml_regrep.xsd.rim._3.SlotType1> documentSlots,
-            String patientInfoName) {
+        String patientInfoName) {
         String slotValue = null;
 
-        for (oasis.names.tc.ebxml_regrep.xsd.rim._3.SlotType1 slot : documentSlots) {
-            if (DocRepoConstants.XDS_SOURCE_PATIENT_INFO_SLOT.equals(slot.getName())) {
+        for (SlotType1 slot : documentSlots) {
+            if (slot.getName().equals(DocRepoConstants.XDS_SOURCE_PATIENT_INFO_SLOT)) {
```

Comment:
yes you are right. Name is a required field in oasis.names.tc.ebxml_regrep.xsd.rim._3.SlotType, so we can just check for slot!=null